### ISSUE TITLE
This mod changes how world generation functions, making uninstallation a nightmare

### ISSUE THREAD

**Issue by megatrook** [spoorn/sodium-forge#170]
title. we uninstalled Sodium off of our Crucial 2 server because it was causing issues with other mods and functionalities (investigating a memory leak and ) and now we have funny/ugly chunk borders :/
![image](https://user-images.githubusercontent.com/77197776/132224456-de523f2f-e7ba-4512-8e8a-489cbf3e6738.png)

**Comment by FlameyFaucet** [comment]
It might be lithium :trollface:
You didn't remove any other worldgen mods did you?

**Comment by megatrook** [comment]
nope, just sodium. it was, in fact, the lithium part of halogen, from what we could tell, which we didnt even realize was a thing. it does cause some issues and incompatibilities though if someone who isn't really savvy with config files, for example, installs sodium but then wants to continue playing on an old world

**Comment by Pannoniae** [comment]
Maybe it's not documented enough but in the config file there's an option you can enable for vanilla seed parity.
